releases/release_notes_09_2022.md

```diff
@@ -0,0 +1,282 @@
+
+```
+kubectl apply  -f https://raw.githubusercontent.com/confidential-containers/operator/main/config/samples/ccruntime.yaml
+```
+
+Check that the ccruntime was created successfully: 
+```
+kubectl get ccruntimes
+```
+Output: 
+``` 
+NAME               AGE
+ccruntime-sample   5s
+```
+
+Use the following command to observe the details of the CR yaml::
+```
+kubectl get ccruntimes ccruntime-sample -o yaml | less
+```
+
+Note that we are using  runtimeName: kataataame: kata
+
+If we were use enclave-cc for example we would observe that runtimeName: enclave-cc
+
+Once we also create the custom resource the validation will show us 2 additional pods created:
+```
+kubectl get pods -n confidential-containers-system
+```
+Output: 
+```
+NAME                                              READY   STATUS    RESTARTS   AGE
+cc-operator-controller-manager-5df7584679-kffzf   2/2     Running   0          21m
+cc-operator-daemon-install-xz697                  1/1     Running   0          6m45s
+cc-operator-pre-install-daemon-rtdls              1/1     Running   0          7m2s
+```
+
+Once the CR was created you will notice we have multiple runtime classes:
+```
+kubectl get runtimeclass
+```
+Output: 
+```
+NAME            HANDLER         AGE
+kata            kata            9m55s
+kata-clh        kata-clh        9m55s
+kata-clh-tdx    kata-clh-tdx    9m55s
+kata-qemu       kata-qemu       9m55s
+kata-qemu-tdx   kata-qemu-tdx   9m55s
+```
+
+Details on each of the runtime classes:
+
+-- kata - standard kata runtime using the QEMU hypervisor including all COCO building blocks for a non CC HW
+-- kata-clh - standard kata runtime using the cloud hypervisor including all COCO building blocks for a non CC HW
+-- kata-clh-tdx - using the Cloud Hypervisor, with TD-Shim, and support for Intel TDX CC HW
+-- kata-qemu - same as kata
+-- kata-qemu-tdx - using QEMU, with TDVF, and support for Intel TDX CC HW
+-- * *TBD: we need to add the SEV runtimes as well* *
+
+
+# Post installation configuration 
+* *TBD:...* *
+
+# Creating a workload 
+## Creating a sample COCO workload 
+
+The first workload we create will show how the COCO building blocks work together without encryption or CC HW support (which will be demonstrated in later workloads). 
+
+A key point when working on COCO is to ensure that the container images get downloaded inside the VM and not on the host.
+
+In our example we will be using the bitnami/nginx image as described in the following yaml:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: nginx
+  name: nginx
+spec:
+  containers:
+  - image: bitnami/nginx:1.22.0
+    name: nginx
+  dnsPolicy: ClusterFirst
+  runtimeClassName: kata
+```
+
+For verifying that the container image doesn’t exist on the host you should log into the k8s node and ensure the following command returns an empty result:
+```
+root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
```

Comment:
The potentially contentious, but it may be worth using the [`$ ` prefix for all commands the user should type](https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks) so that if a line doesn't start with that prefix, it's clear it's output.

But here, I wonder if a simple `sudo` would be clearer than the slightly scary prompt? :)

```suggestion
$ sudo crictl -r unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
```

Same comment for all other commands and output.